Airports/EGLC/Sector.txt

```diff
@@ -42,140 +42,261 @@ DISPLAY:LCTWR:Only LCAPP:LCTWR
+DISPLAY:Only Thames:Only Thames:London TC SE
+DISPLAY:Only LCAPP:Only Thames:London TC JACKO
+DISPLAY:Only LCAPP:Only Thames:London TC SE
+COORD:N051.39.08.000:E000.14.16.000
+COORD:N051.30.09.004:E000.08.26.570
 
 SECTORLINE:THA8
-DISPLAY:Thames:Thames:London TC TIMBA
-COORD:N051.13.59.090:E000.19.34.398
-COORD:N051.13.47.205:E000.20.22.651
-COORD:N051.13.28.439:E000.23.24.209
-COORD:N051.19.46.971:E000.39.58.068
+DISPLAY:Only Thames:Only Thames:London TC JACKO
+DISPLAY:Only Thames:Only Thames:London TC SE
+DISPLAY:Only LCAPP:Only Thames:London TC JACKO
+DISPLAY:Only LCAPP:Only Thames:London TC SE
+COORD:N051.38.46.000:E000.09.06.000
+COORD:N051.39.08.000:E000.14.16.000
 
 SECTORLINE:THA9
-COORD:N051.19.46.971:E000.39.58.068
-COORD:N051.22.46.750:E000.37.37.000
+DISPLAY:Only Thames:Only Thames:London TC JACKO
+DISPLAY:Only Thames:Only Thames:London TC SE
+DISPLAY:Only LCAPP:Only Thames:London TC JACKO
+DISPLAY:Only LCAPP:Only Thames:London TC SE
+COORD:N051.39.08.000:E000.14.16.000
+COORD:N051.40.00.855:E000.25.55.593 ;Near EVNAS
 
 SECTORLINE:THA10
-COORD:N051.31.58.642:E000.08.52.645
-COORD:N051.31.34.148:E000.23.35.723
-COORD:N051.22.46.750:E000.37.37.000
+DISPLAY:Only Thames:Only Thames:London TC JACKO
+DISPLAY:Only Thames:Only Thames:London TC SE
+DISPLAY:Only LCAPP:Only Thames:London TC JACKO
+DISPLAY:Only LCAPP:Only Thames:London TC SE
+COORD:N051.40.00.855:E000.25.55.593 ;Near EVNAS
+COORD:N051.31.46.000:E000.24.00.000
+COORD:N051.29.34.117:E000.25.55.256
 
 SECTORLINE:THA11
-COORD:N051.37.43.633:E000.26.21.852
-COORD:N051.22.46.750:E000.37.37.000
+DISPLAY:Only Thames:Only Thames:London TC JACKO
+DISPLAY:Only Thames:Only Thames:London TC SE
+DISPLAY:Only LCAPP:Only Thames:London TC JACKO
+DISPLAY:Only LCAPP:Only Thames:London TC SE
+COORD:N051.29.34.117:E000.25.55.256 ;Between TOPDU and NEPNA
+COORD:N051.29.40.467:E000.22.55.977 ;Near NEPNA
 
 SECTORLINE:THA12
-DISPLAY:Thames:Thames:London TC TIMBA
-COORD:N051.35.58.660:E000.09.28.494
-COORD:N051.37.49.179:E000.25.24.591
-COORD:N051.37.43.633:E000.26.21.852
+DISPLAY:Only Thames:Only Thames:London TC JACKO
+DISPLAY:Only Thames:Only Thames:London TC SE
+DISPLAY:Only LCAPP:Only Thames:London TC JACKO
+DISPLAY:Only LCAPP:Only Thames:London TC SE
+COORD:N051.30.09.004:E000.08.26.570
+COORD:N051.29.40.467:E000.22.55.977
 
 SECTORLINE:THA13
-DISPLAY:Thames:Thames:London TC TIMBA
-COORD:N051.37.43.633:E000.26.21.852
-COORD:N051.37.03.250:E000.34.09.595
-COORD:N051.44.41.185:E000.51.06.789
-COORD:N051.46.22.550:E001.08.33.160
+DISPLAY:Only Thames:Only Thames:London TC JACKO
+DISPLAY:Only Thames:Only Thames:London TC SE
+DISPLAY:Only LCAPP:Only Thames:London TC JACKO
+DISPLAY:Only LCAPP:Only Thames:London TC SE
+COORD:N051.29.40.467:E000.22.55.977
+COORD:N051.24.52.079:E000.27.07.703
+COORD:N051.24.48.805:E000.30.05.615
 
-SECTORLINE:THA14
-COORD:N051.46.22.550:E001.08.33.160
-COORD:N051.37.25.933:E001.16.31.502
-COORD:N051.35.30.217:E001.08.59.862
+SECTORLINE:THA14 ;Unused
```

Comment:
It looks as if we've unified the airspace a little more than real world, so I didn't end up needing this line. But it would be a bit of a hassle to rename them all, so I'm leaving it in, perhaps one day it having a use...